### PR TITLE
[2] Send drive location to user via kafka

### PR DESCRIPTION
**Pull request by Yurchenko-Anton**

### CODE REVIEW

File: src/main/java/com/example/action/config/KafkaConfig.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import org.springframework.kafka.config.TopicBuilder;
+import org.springframework.kafka.support.KafkaHeaders;
+import org.springframework.messaging.handler.annotation.Header;
+import org.springframework.messaging.handler.annotation.Payload;
+
+@Configuration
+@RequiredArgsConstructor
+public class KafkaConfig {
+
+    public static final String CONSUMER_DRIVER_LOCATION_TOPIC = "consumer-driver-location-topic";
```
Comment (asinchuk) [inline src/main/java/com/example/action/config/KafkaConfig.java:18]: move to properties please and shorten topic name

File: src/main/java/com/example/action/config/KafkaConfig.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    public static final String GROUP_ID = "group";
```
Comment (asinchuk) [inline src/main/java/com/example/action/config/KafkaConfig.java:19]: same

File: src/main/java/com/example/action/config/KafkaConfig.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    public static final String PRODUCER_DRIVER_LOCATION_TO_CLIENT_TOPIC = "producer-driver-location-to-client-topic";
```
Comment (asinchuk) [inline src/main/java/com/example/action/config/KafkaConfig.java:20]: same

File: src/main/java/com/example/action/service/DriverLocationProcessor.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+            Order order = orderDriverRepository.findOrderByDriverId(Long.parseLong(id)).getBody();
+            if (order != null) {
+                sendMessage(order.getPassengerId().toString(), "Driver : " + id + " on the street : " + msg);
+            } else log.info("Driver " + id + " dosen't have an active order");
+        } catch (Exception e) {
+            log.error(e.toString());
+        }
+    }
+
+    public void sendMessage(String passengerId, String msg) {
```
Comment (asinchuk) [inline src/main/java/com/example/action/service/DriverLocationProcessor.java:31]: inline method name with processDriverLocationMessage

File: src/main/java/com/example/action/config/KafkaConfig.java (hunk 3 above)
Comment (Yurchenko-Anton) [reply src/main/java/com/example/action/config/KafkaConfig.java:20]: agree to leave as it is

File: src/main/java/com/example/action/config/KafkaConfig.java (hunk 2 above)
Comment (Yurchenko-Anton) [reply src/main/java/com/example/action/config/KafkaConfig.java:19]: agree to leave as it is

File: src/main/java/com/example/action/config/KafkaConfig.java (hunk 1 above)
Comment (Yurchenko-Anton) [reply src/main/java/com/example/action/config/KafkaConfig.java:18]: agree to leave as it is

File: src/main/java/com/example/action/service/DriverLocationProcessor.java (hunk 4 above)
Comment (Yurchenko-Anton) [reply src/main/java/com/example/action/service/DriverLocationProcessor.java]: fixed

**Review by asinchuk — APPROVED** [review]

**Pull request merged**